lib/engine/game/g_18_india/step/home_track.rb

```diff
@@ -43,12 +43,39 @@ def process_lay_tile(action)
               connected: false,
               extra_action: true
             )
+
             @round.pending_tokens.shift
           end
 
           def process_place_token(action)
             super
             tile = action.city.tile
+            # apply "other" OO token once first to operate is applied if floated
+            if action.entity.name == 'NWR' && tile.name == '235'
+              corporation = @game.corporations.find { |c| c.name == 'SPD' }
```

Comment:
There's a method to find corporations easily:
```suggestion
              corporation = @game.corporation_by_id('SPD')
```